vendor/github.com/IBM/secret-utils-lib/pkg/authenticator/pod-identity.go

```diff
@@ -56,6 +58,18 @@ func (ca *ComputeIdentityAuthenticator) GetToken(freshTokenRequired bool) (strin
 	tokenResponse, err := ca.authenticator.RequestToken()
 	if err != nil {
 		ca.logger.Error("Error fetching fresh token", zap.Error(err))
+		if !strings.Contains(strings.ToLower(err.Error()), "timeout") {
```

Comment:
looks same code is https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/pull/106/files#diff-7adde7e8cbdf8fb4001efd492c7202ad64e641be33ea637f89206f709d6b1d59R63 as well, can you make a method and re-use it

Reply:
As these are vendor file changes , code must be fixed from the library side. Internal issue is opened to address the same.

Reply:
As these are vendor file changes , code must be fixed from the library side. Internal issue is opened to address the same.